jobmanager/WorkerThread.h

```diff
@@ -15,6 +15,20 @@ class WorkerThread
 
     static double getWaitTime(); // ms
     static double getWorkTime(); // ms
+
+    /**
+     * @brief Checks whether current thread is a WorkerThread or not
+     * @return true if current thread is a worker thread; otherwise, false.
+     */
+    static bool isWorkerThread();
```

Comment:
The method name in this case is excellent.
It already says everything that is in the comment.